wallet/src/ui/app/pages/transfer-nft/validation.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as Yup from 'yup';
+
+import { SUI_ADDRESS_VALIDATION } from '_components/address-input/validation';
+import {
+    DEFAULT_GAS_BUDGET_FOR_TRANSFER,
+    GAS_TYPE_ARG,
+    GAS_SYMBOL,
+} from '_redux/slices/sui-objects/Coin';
+
+import type { FormatNumberOptions, IntlShape } from 'react-intl';
+
+export function createValidationSchema(
+    coinType: string,
+    coinBalance: bigint,
+    coinSymbol: string,
+    gasBalance: bigint,
+    totalGasCoins: number,
+    intl: IntlShape,
+    formatOptions: FormatNumberOptions
+) {
+    return Yup.object({
+        to: SUI_ADDRESS_VALIDATION,
+        amount: Yup.number()
```

Comment:
I think the current validation works well for this.

Reply:
I added some examples but also the min/max checks are not necessary since we only use one value [here](https://github.com/MystenLabs/sui/pull/2778/files#diff-a858a650f16e6c4b94899a78212fa12026187404f134b00582134ce6a6635d40R28) and never changed it